seacatauth/external_login/handler.py

```diff
@@ -67,12 +67,23 @@ async def login(self, request):
 		sub = str(sub)
 
 		# Get credentials by sub
-		credentials = await self.CredentialsService.get_by_external_login_sub(login_provider_type, sub)
-		if credentials is None:
+		try:
+			el_credentials = await self.ExternalLoginService.get(login_provider_type, sub)
+			credentials_id = el_credentials["cid"]
+		except KeyError:
+			credentials_id = None
+		if credentials_id is None:
+			try:
+				credentials = await self.CredentialsService.get_by_external_login_sub(login_provider_type, sub)
+				# TODO: Migrate external login data to the dedicated collection?
```

Comment:
Is this TODO still valid?

Reply:
TODO: Do research if it's used anywhere. If it is, extend the TODO and implement per-user migration.